src/appleseed/renderer/modeling/frame/frame.cpp

```diff
@@ -855,22 +623,6 @@ DictionaryArray FrameFactory::get_input_metadata()
 
     metadata.push_back(
         Dictionary()
-            .insert("name", "pixel_format")
```

Comment:
We should probably introduce a migration step to remove the old parameters from frame entities.